### DISCUSSION TITLE
Starred  repo

### DISCUSSION THREAD

**Discussion by kailash16** [community/community#6223] [Mobile]
Couldn't able to find my starred repo in ios.

**Comment by taki-on** [comment]
You should be able to access to your starred repos via:
Tap on `Profile` on the bottom tab bar -> Tap on `Starred` with the yellow icon.

I am wondering if you still do not see any starred repos?

**Reply by kailash16** [reply]
Yes, in iPad app I couldn't able to see. But it is visible in phone app
